### ISSUE TITLE
Blog Theme & Design

### ISSUE THREAD

**Issue by laurburke** [scikit-learn/blog#67]
Creating a new issue to log design requests, suggestions, and decisions. 
CC: @reshamas

***
#### Production
V1: [blog.scikit-learn.org](https://blog.scikit-learn.org)

***

### Theme Options
Current Theme
- [Minimal Mistakes](https://mmistakes.github.io/minimal-mistakes/)

Other Themes
- Other options for Jekyll blogs: https://jekyllthemes.io/jekyll-blog-themes
- Pelican: https://blog.getpelican.com
- Hugo themes: https://themes.gohugo.io

### Reference
Other Blogs
- Jupyter Blog (on Medium): https://blog.jupyter.org/archive
- Inria blog: https://scikit-learn.fondation-inria.fr/news/
- Jupyter calendar: https://docs.jupyter.org/en/latest/community/content-community.html#jupyter-community-meetings
- Data Umbrella (paid theme): https://blog.dataumbrella.org/
- Pandas Blog: https://dev.pandas.io/pandas-blog/
- Brit + Co: https://www.brit.co/

### Current Design
Menus
- Header: scikit-learn.org, Calendar, Resources
    - Dropdown: Install, User Guide, API, Examples
- Sidebars
    - Social Media links
    - Archives: Category, Tag, Year
    - scikit-learn: About Us, FAQ, Code of Conduct
    - Mailing List: Subscribe, Archive
- Footer: Social Media links, Feed

Home Page
- Hero Banner: featured image, text 
- Recent Posts Queue
    - _Request:_ Display posts' featured image
    - Author, read time
- Twitter sidebar (currently hidden if viewer using adblock)

Posts
- Display authors
- Display Category, Tag

Pages
- Calendar: embedded public scikit-learn event calendar
- Resources

***
### Requests & Suggestions
- Request: show earlier posts to viewers
  - We currently have [Category](https://blog.scikit-learn.org/categories/), [Tag](https://blog.scikit-learn.org/tags/) and [Year](https://blog.scikit-learn.org/years/) archives. If you have suggestions of an alternative way to display earlier posts, please comment and provide a reference link if available. 
- Request: display the number of posts
  - Where should we display blog statistics?
  - Number of posts by category/tag/year? Total number?
- Request: display posts' featured image in recent/archived posts queue

**Comment by jnothman** [comment]
This is a great start!

Aesthetics that are a bit uncomfortable for me in Minimal Mistakes:
* Side bar clutters the view while reading. If we keep it, it could disappear when the user scrolls down.
* I find the text column width very small on a wide window
  ![image](https://user-images.githubusercontent.com/78827/151874386-ced3c794-841d-4008-9d91-09a91684739e.png)
* I do like the homepage with images like in data umbrella.

**Comment by jjerphan** [comment]
To me, this is already a good result for a V1: thank you, @reshamas and @laurburke. :+1: 

A few points that can be improved:
 - a few HTML elements can be fixed based on [W3C validation](https://validator.w3.org/nu/?doc=http%3A%2F%2Fscikit-learn.org%2Fblog%2F). This is probably important for SEO.
  - I find the Sans Serif font a bit hard to read.

**Comment by adrinjalali** [comment]
I agree with what's raised already, plus, it'd be nice if the right sidebar (the tweets) looked better when certain cookies are blocked by plugins. Ideally, it could say "this part is not available due to you blocking xyz".

**Comment by laurburke** [comment]
Of the themes linked above, are there any that stand out? In our last meeting, many seemed to like the [theme Data Umbrella uses for their blog](https://blog.dataumbrella.org/).

**Comment by adrinjalali** [comment]
We can use our NumFocus funds to get the theme with no delay. Once you've made a decision, if there's a need to buy the theme, can do that.

**Comment by laurburke** [comment]
Thank you to everyone for your suggestions!
